web-api/src/genericHandler.js

```diff
@@ -97,6 +97,16 @@ exports.genericHandler = (awsEvent, cb, options = {}) => {
         });
       }
 
+      const responsePayload = JSON.stringify(returnResults);
+      const responsePayloadSizeBytes = responsePayload.length * 2;
```

Comment:
Just curious why the `* 2;`? 🤔

Reply:
removed, using the Buffer.byteLength approach instead, thank you!